_posts/2.6/2021-09-22-dev.md

```diff
@@ -587,6 +587,25 @@ sbt debian:packageBin
 dpkg -i ./target/bbb-fsesl-akka_<tab>.deb
 ```
 
+# Developing bbb-export-annotations
+
+You can manually run the service.
+When running, it'll process export jobs stored in Redis containing the state of the whiteboard at the time of the request.
+To install the npm dependencies, run:
+
+```
+cd ~/dev/bigbluebutton/bbb-export-annotations
+npm install
```

Comment:
Do you need to first stop (ensure it's not running) the packaged bbb-export-annotations? Something like `sudo systemctl stop bbb-export-annotations`?

Reply:
Yes, thanks for pointing that out